easybuild/easyconfigs/s/SimNIBS/SimNIBS-4.5.0_setup.py-fix.patch

```diff
@@ -0,0 +1,57 @@
+--- setup.py.orig	2025-08-27 09:28:29.000000000 +0200
```

Comment:
```suggestion
Fix for setup.py to use EB dependecies CGAL and allow building outside of CONDA
--- setup.py.orig	2025-08-27 09:28:29.000000000 +0200
```

Fix for CI, it is good to have a short description of what a patch is doing for people reading it in the future

You can also add your preferred `Author: ...` line if you want in the description